pkg/server/container_create.go

```diff
@@ -440,9 +441,9 @@ func setOCICapabilities(g *generate.Generator, capabilities *runtime.Capability,
 }
 
 // setOCINamespaces sets namespaces.
-func setOCINamespaces(g *generate.Generator, namespaces *runtime.NamespaceOption, sandboxPid uint32) {
-	g.AddOrReplaceLinuxNamespace(string(runtimespec.NetworkNamespace), getNetworkNamespace(sandboxPid)) // nolint: errcheck
-	g.AddOrReplaceLinuxNamespace(string(runtimespec.IPCNamespace), getIPCNamespace(sandboxPid))         // nolint: errcheck
-	g.AddOrReplaceLinuxNamespace(string(runtimespec.UTSNamespace), getUTSNamespace(sandboxPid))         // nolint: errcheck
-	g.AddOrReplaceLinuxNamespace(string(runtimespec.PIDNamespace), getPIDNamespace(sandboxPid))         // nolint: errcheck
+func setOCINamespaces(g *generate.Generator, namespaces *runtime.NamespaceOption, sandbox *sandboxstore.Sandbox) {
+	g.AddOrReplaceLinuxNamespace(string(runtimespec.NetworkNamespace), sandbox.NetNS)            // nolint: errcheck
```

Comment:
Whether we could continue using `getNetworkNamespace(sandboxPid)` here? Need to think about it.